internal/shim-sgx/Cargo.toml

```
@@ -17,7 +17,7 @@ dbg = []
 gdbstub_arch = { version = "0.1.1" , default-features = false, optional = true }
 gdbstub = { version = "0.5.0" , default-features = false, optional = true }
 goblin = { version = "0.5", default-features = false, features = [ "elf64" ] }
-primordial = { version = "^0.4.0", features = ["const-default"] }
+primordial = { git = "https://github.com/enarx/primordial", branch = "main", features = ["const-default"] }
```

Please use git hash references, so, that if the remote branch gets updated, the enarx builds don't break.
```suggestion
primordial = { git = "https://github.com/enarx/primordial", rev = "b606b9e472e2238384b003f12fdada077e04d2fe", features = ["const-default"] }
```